src/server.c

```diff
@@ -7443,7 +7443,7 @@ int main(int argc, char **argv) {
             if (listener->ct == NULL)
                 continue;
 
-            serverLog(LL_NOTICE,"Ready to accept connections %s", listener->ct->get_type(NULL));
+            serverLog(LL_NOTICE,"Ready to accept connections via %s", listener->ct->get_type(NULL));
```

Comment:
IMHO, the original log felt fine and no ambiguity.

Reply:
@sundb I am happy to revert on this but here is my reasoning for the change:

- The previous message is missing a preposition, making it feel incomplete.
- The first message is ambiguous.

The suggestion uses “via,” which is a proper preposition to indicate the medium of connection, improving clarity.

Reply:
I prefer to revert this change, this log exists very long time, some users may check this log to confirm if server is running, it may fail after changing, so i think we should avoid this change

Reply:
@sundb @ShooterIT Reverting 👌🏾